docs/src/modules/spring/pages/views.adoc

```diff
@@ -26,7 +26,7 @@ Consider an example of a Customer Registry service with a `customer` Value Entit
 <<_define_the_view>> for a service that selects customers by name and associates a table name with the View. The table is created and used by Kalix to store the View.
 
 
-This example assumes the following `customer` exists
+This example assumes the following `Customer` exists:
```

Comment:
Maybe a link to sample/doc on how to create such an Entity?

Reply:
thanks. added now.